src/citrine/resources/file_link.py

```diff
@@ -648,6 +675,10 @@ def read(self, *, file_link: Union[str, UUID, FileLink]):
         """
         file_link = self._resolve_file_link(file_link)
 
+        if self._is_local_url(file_link.url):  # Read the local file
+            path = Path(unquote_plus(urlparse(file_link.url).path))
+            return path.read_bytes()
+
```

Comment:
If a local path is provided, `requests` cannot support it.  Rather than create local v. remote branches throughout, seems better to just have `read` be agnostic.